ebusd/README.md

```
@@ -8,9 +8,27 @@ See [repository readme](https://github.com/LukasGrebe/ha-addons#how-to-install)
 
 See [docs](https://github.com/LukasGrebe/ha-addons/blob/main/ebusd/DOCS.md#how-to-run-ebusd) on how to run ebusd in supervisor.
 
-**If you have questions or feedback please**
-- via the [Home Assistant Forums Topic](https://community.home-assistant.io/t/an-ebusd-add-on/344852)
-- via Issues and pull requests in the Github repository
+## Support
+
+**Issues in Configuration and Usage**
+Lots of issues result from incomplete [ebusd configuration](https://github.com/john30/ebusd/wiki/4.-Configuration) files. This project only runs ebusd, configurationfiles are **not** managed by this project. Please see the offical [ebusd project](https://ebusd.eu) and [community](https://github.com/john30/ebusd/discussions) for more information. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)